sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/SamplesArguments.java

```diff
@@ -0,0 +1,69 @@
+package com.azure.digitaltwins.core;
+
+import org.apache.commons.cli.*;
+
+public class SamplesArguments {
+
+    private final String DIGITALTWINS_URL = "DigitalTwinsEndpoint";
+    private final String TENANT_ID = "tenantId";
+    private final String CLIENT_ID = "clientId";
+    private final String CLIENT_SECRET = "clientSecret";
+
+    private String digitalTwinUrl;
+    private String tenantId;
+    private String clientId;
+    private String clientSecret;
+
+    public SamplesArguments(String[] args) {
+        Options options = new Options();
+        Option input = new Option("d", DIGITALTWINS_URL, false, "DigitalTwins endpoint URL");
+        Option tenantId = new Option("t", TENANT_ID, false, "AAD Tenant Id");
+        Option clientId = new Option("c", CLIENT_ID, false, "AAD Client Id");
+        Option clientSecret = new Option("s", CLIENT_SECRET, false, "AAD Client Secret");
+
+        input.setRequired(true);
```

Comment:
Ah, is there no way to set it inline during initialization? Or some annotation perhaps?

Reply:
unfortunately not :(